src/main/java/io/dashboardhub/pipelinedashboard/listener/AuthenticationListener.java

```diff
@@ -7,6 +7,8 @@
 import org.springframework.security.authentication.event.InteractiveAuthenticationSuccessEvent;
 import org.springframework.stereotype.Component;
 
+import java.util.Date;
```

Comment:
avoid using Date class wherever possible. Use the Java8 time/date api instead - eg LocalDateTime